src/Codeception/Module/Laravel5.php

```diff
@@ -1167,6 +1167,63 @@ public function haveMultiple($model, $times, $attributes = [], $name = 'default'
             $this->fail("Could not create model: \n\n" . get_class($e) . "\n\n" . $e->getMessage());
         }
     }
+    
+    /**
+     * Use Lumen's model factory to make a model instance.
+     * Can only be used with Lumen 5.1 and later.
+     *
+     * ``` php
+     * <?php
+     * $I->make('App\User');
+     * $I->make('App\User', ['name' => 'John Doe']);
+     * $I->make('App\User', [], 'admin');
+     * ?>
+     * ```
+     *
+     * @see https://lumen.laravel.com/docs/master/testing#model-factories
```

Comment:
You left Lumen links in Laravel docblocks. Please fix them.
markdown files are regenerated by release process, so the changes you made there will be lost if they aren't the same as the changes in code files.

Reply:
Ok. Thank I'll fix them. 
For your previous comment: I've added that chamge to the DataFactory, but for Laravel it utilizes their factories.